test/cri-containerd/stats_test.go

```diff
@@ -354,3 +365,76 @@ func Test_ContainerStats_List_ContainerID(t *testing.T) {
 		})
 	}
 }
+
+func Test_SandboxStats_WorkingSet_PhysicallyBacked(t *testing.T) {
```

Comment:
You should try to add a test for non-physically backed to verify that working set size is as expected.